src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/EnvelopeMarkAsDeletedService.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.services;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.EnvelopeJdbcRepository;
+
+import java.util.UUID;
+
+@Service
+public class EnvelopeMarkAsDeletedService {
+    private static final Logger log = LoggerFactory.getLogger(EnvelopeMarkAsDeletedService.class);
+
+    private final EnvelopeJdbcRepository envelopeJdbcRepository;
+
+    public EnvelopeMarkAsDeletedService(EnvelopeJdbcRepository envelopeJdbcRepository) {
+        this.envelopeJdbcRepository = envelopeJdbcRepository;
+    }
+
+    public void markEnvelopeAsDeleted(UUID envelopeId, String loggingContext) {
```

Comment:
do we need to annotate as transactional?

Reply:
Of course, this is what this PR for. Thank you, committed.